### PR TITLE
Feature/sqli table mapping

### PR DESCRIPTION
**Pull request by afeena**
Auto generated query map

### CODE REVIEW

File: sqli_emulator.py [hunk 1]
```diff
@@ -17,15 +17,41 @@ def setup_db(self):
...
+                try:
+                    for row in c.execute(query):
+                        columns.append(row[1])
+                    query_map[table] = columns
+                except sqlite3.OperationalError as e:
+                    print(e)
+        return query_map
+
+    @asyncio.coroutine
+    def map_query(self, query):
```
Comment (glaslos) [inline sqli_emulator.py]: How often do you call this?

File: tests/test_sqli.py [hunk 2]
```diff
@@ -8,7 +8,17 @@
 class SqliTest(unittest.TestCase):
     def setUp(self):
         open('/tmp/test.db', 'a').close()
+
+        @asyncio.coroutine
+        def foo():
```
Comment (glaslos) [inline tests/test_sqli.py]: foo? :)

Comment (afeena) [inline tests/test_sqli.py]: Naming makes me cry :)

Comment (glaslos) [inline tests/test_sqli.py]: You are not the only one :)

**Pull request merged**